src/wrappers/themis/java/com/cossacklabs/themis/SecureCell.java

```diff
@@ -619,10 +619,10 @@ public SecureCell(int mode) {
 		if (mode < MODE_SEAL || mode > MODE_CONTEXT_IMPRINT) {
 			throw new InvalidArgumentException("invalid mode");
 		}
-		
+
```

Comment:
Saving the extra space, huh?

Reply:
I have trained my editor to treat trailing whitespace as evil. I guess I'll revert this particular thing to avoid diff noise.